packages/gestalt/src/Modal.js

```diff
@@ -31,6 +31,7 @@ const ESCAPE_KEY_CODE = 27;
 
 function Backdrop({ children }: { children?: React.Node }) {
   return (
+    // eslint-disable-next-line react/jsx-fragments
     <React.Fragment>
```

Comment:
Why use `<React.Fragment>` here (with lint disable) and `<>` later?

Reply:
@btraut updated - thanks for spotting.
Already had this change locally but didn't push it up.